lite-pm/pages/components/AppPage.js

```diff
@@ -122,7 +110,16 @@ class AppPage extends Component {
         }
       }
 
-      this.setState({ Member: members, todoTasks: todos });
+      this.setState(
+        {
+          Member: members,
+          todoTasks: todos,
+          completedTasks: completed,
+        },
+        () => {
+          console.log(this.state.todoTasks);
```

Comment:
you can leave some of the logs as long as there is an explanation as to what it is, e.g. `console.log("todo tasks after this or that: ", this.state.todoTasks);`